public/server/db/hsd.json

```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Happy Sunday"
```

Comment:
I know they may be different variations to abbreviations, but is Happy Sunday not usually `HS`?

Reply:
> I know they may be different variations to abbreviations, but is Happy Sunday not usually `HS`?

I'm not so sure about what it's being called on your end. But here, it's commonly called that.

Reply:
> I know they may be different variations to abbreviations, but is Happy Sunday not usually `HS`?

I'm not so sure about what it's being called on your end. But here, it's commonly called that.